docs/conditions.md

```diff
@@ -35,9 +35,9 @@ following fields:
 
 The `check` field is required. You define a single check to define the body of a `Condition`. The 
 check must specify a container image that adheres to the [container contract](./container-contract.md). 
-The container image runs till completion. The container must exit successfully i.e. with an exit code 0 
-for the condition evaluation to be successful. All other exit codes are considered to be a condition check
-failure.
+Similar to `Task` the check also supports [`script` mode](./tasks.md#step-script). The container image runs 
```

Comment:
I think it would make sense to say something along the lines of:  "The check must specify a _Step_ (link to Step docs)" as opposed to keeping references to the container image/contract.